docusaurus.config.js

```diff
@@ -200,7 +201,6 @@ const config = {
     }),
 
   plugins: [
-    '@orama/plugin-docusaurus',
```

Comment:
Yes I also enoutered that issue where it says there are some missing items. I was able to pinpoint the issue ...I believe it was due a certain command copying docs from say <version-folder>/* into <version-folder>/docs . so when you tun the build website again which is designed to look into <version-folder>/*  and not <version-folder>/docs which now causes a build error since the files in <version-folder>/docs were not updated. Will recreate this and get back to you

Reply:
The fix has been released!